test/timeout.bats

```diff
@@ -23,19 +27,32 @@ fi
 $CONMON_BINARY \$@
 EOF
 	chmod +x "$TESTDIR/tmp_conmon"
+
+	CONTAINER_CONMON="$TESTDIR/tmp_conmon"
+}
+
+# Allow cri-o to catch up. The sleep here should be less than
+# resourcestore.sleepTimeBeforeCleanup but enough for cri-o to
+# finish processing cancelled crictl create/runp.
+function wait_create() {
+	sleep 30s
+}
+
+# Allow cri-o to catch up and clean the state of a pod/container.
+# The sleep here should be > 2 * resourcestore.sleepTimeBeforeCleanup.
+function wait_clear() {
```

Comment:
maybe wait_clean? we don't use the verb clear anywhere really

Reply:
right

Reply:
updated